src/presentation/http/router/join.test.ts

```diff
@@ -42,25 +76,52 @@ describe('Join API', () => {
     });
 
     test('Returns 200 when user is added to the team', async () => {
-      const hash = 'Hzh2hy4igf';
-      const userId = 1;
-      const accessToken = global.auth(userId);
+      const invitationHash = 'Hzh2hy4igf';
```

Comment:
I think the variable should be declared below, before `insertNoteSettings`, because otherwise it is lost